tests/validate/xcl_vcu_test/src/host.cpp

```diff
@@ -80,8 +80,14 @@ int main(int argc, char** argv) {
   // Harcoding the number of processes/instances 
   int ret = vcu_dec_test(binaryFile.c_str(), TEST_INSTANCE_ID, device_index);
   if (ret != TRUE) {
-    std::cout << "TEST FAILED\n";
-    return EXIT_FAILURE;
+    if (ret == FALSE) {
```

Comment:
we should simplify this:
if(ret == FALSE)...
else if (ret!= not_SUPP)...

Reply:
Done.